agent/fw_drupal8.c

```diff
@@ -71,6 +71,48 @@ static void nr_drupal8_add_method_callback(const zend_class_entry* ce,
   }
 }
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+    && !defined OVERWRITE_ZEND_EXECUTE_DATA
+static void nr_drupal8_add_method_callback_before_after_clean(
```

Comment:
This is essentially a copy of https://github.com/newrelic/newrelic-php-agent/blob/main/agent/fw_drupal8.c#L32 that instead calls the underlying before_after_clean wrapping function